htdocs/website/index.php

```diff
@@ -4288,10 +4289,68 @@ function switchEditorOnline(forceenable)
 
 	// Example/templates of page
 	if ($action == 'createcontainer') {
+		$formmail = new FormMail($db);
+		$formmail->withaiprompt = 'html';
+
+
 		print '<tr><td class="titlefield fieldrequired">';
```

Comment:
Add class "tdtop" so title will remain at top, even when the line increase after selecting the Fille layaout or Fill with AI.